shell/src/test/java/alluxio/cli/LogLevelTest.java

```diff
@@ -25,17 +25,17 @@
 import alluxio.conf.PropertyKey;
 import alluxio.job.wire.JobWorkerHealth;
 import alluxio.master.MasterInquireClient;
+import alluxio.master.MasterInquireClient.Factory;
```

Comment:
Avoid importing an inner class with a generic name: there are other `Factory` classes, e.g. `JobMasterClient.Factory`, used in this test.